share/functions/__fish_print_xdg_mimetypes.fish

```diff
@@ -1,3 +1,3 @@
 function __fish_print_xdg_mimetypes --description 'Print XDG mime types'
-    cat {(__fish_print_xdg_applications_directories)}/mimeinfo.cache ^/dev/null | string match -v '[MIME Cache]' | string replace = \t
+    cat {__fish_print_xdg_applications_directories}/mimeinfo.cache ^/dev/null | string match -v '[MIME Cache]' | string replace = \t
```

Comment:
Did you test this? Because it won't work now :smile: "Braces" (sometimes called "curly brackets") are the curly variant. Parentheses are the round variant. You need the parentheses to indicate a subcommand should be run and its output captured. So you removed the wrong pair of punctuation marks. I'll fix it when I merge it. Just a reminder to always double-check that a seemingly trivial change doesn't introduce a problem.